### PR TITLE
Login register styles

### PR DESCRIPTION
**Pull request by jtobin321**
Added modal with form for users to update their password if they've forgotten it. This is only front end work, back end logic needs to be added to handle the request.

### CODE REVIEW

File: client/src/views/Login/Login.js [hunk 1]
```diff
@@ -180,9 +196,39 @@ const Login = ({ ...props }) => {
...
+                          type="email"
+                          fullWidth
+                        />
+                      </DialogContent>
+                      <DialogActions>
+                        <Button onClick={handleClose} color="primary">
+                          Cancel
+                        </Button>
+                        <Button onClick={handleClose} color="primary">
+                          Submit
```
Comment (aglasswala) [inline client/src/views/Login/Login.js]: I would just change this to `Send` and that's it. I'll talk to Cristian about adding stuff to the backend

Comment (jtobin321) [reply client/src/views/Login/Login.js]: It took me a really long time to realize what you were talking about lol, but I changed it. If you don't merge I will merge in the next five minutes.

**Comment by no-vac** [comment]
For reset password can you change the api call to a GET to /u/resetpw/:email ?

That will send them an email with link to hawkhack.io/reset/:token that included their reset token as a parameter. This also needs a form that will take their new password and send make a call to POST /u/resetpw with {email, password, password2} in the body.

For reference:  https://github.com/hawkhack/HawkHack.io-v2/blob/bffe54ea4e06b022f309bc7423657ed5aa022384/routes/api/user.js#L235

**Comment by jtobin321** [comment]
All of the api calls have been updated to match the routes already created. Snackbar styles are not finalized but that can be changed later, for now it's working.

**Comment by aglasswala** [comment]
> For reset password can you change the api call to a GET to /u/resetpw/:email ?
> 
> That will send them an email with link to hawkhack.io/reset/:token that included their reset token as a parameter. This also needs a form that will take their new password and send make a call to POST /u/resetpw with {email, password, password2} in the body.
> 
> For reference:
> 
> https://github.com/hawkhack/HawkHack.io-v2/blob/bffe54ea4e06b022f309bc7423657ed5aa022384/routes/api/user.js#L235

Wouldn't it be safer to send have a token at the end of the URL instead of the email? If I know someone's email, I can force a pw reset

**Pull request merged**